### ISSUE TITLE
form-select[multiple] should not display any indicator/icon

### ISSUE THREAD

**Issue by tkrotoff** [twbs/bootstrap#33591]
Under Bootstrap 4 (4.6.0), custom-select[multiple] does not display the select indicator and the is-valid/invalid icon:

```HTML
<select class="custom-select is-valid" multiple></select>
```

![image](https://user-images.githubusercontent.com/643434/114147971-90f2de80-9919-11eb-98a2-7c24c1b7cd09.png)

Under Bootstrap 5 (5.0.0-beta3), the new form-select[multiple] does display the select indicator and the is-valid/invalid icon:

```HTML
<select class="form-select is-valid" multiple></select>
```

![image](https://user-images.githubusercontent.com/643434/114153910-22fde580-9920-11eb-8023-6fb2c028e789.png)

This is a regression.

**Comment by alecpl** [comment]
Probably fixed by #33411

**Comment by tkrotoff** [comment]
IMHO the PR is wrong/not enough, see my comment here: https://github.com/twbs/bootstrap/pull/33411#issuecomment-816675388
